audio/out/ao_pcm.c

```diff
@@ -86,7 +86,7 @@ static void write_wave_header(struct ao *ao, FILE *fp, uint64_t data_length)
     fput16le(WAV_ID_FORMAT_EXTENSIBLE, fp);
     fput16le(ao->channels.num, fp);
     fput32le(ao->samplerate, fp);
-    fput32le(ao->bps, fp);
+    fput32le(MPCLAMP(ao->bps, 0, UINT32_MAX), fp);
```

Comment:
this will write an invalid file when `ao->bps > UINT32_MAX`, no?

Reply:
`bps > UINT32_MAX` is already invalid, so nothing of value will be lost.

Reply:
I was thinking it could log an error instead of writing a broken file.